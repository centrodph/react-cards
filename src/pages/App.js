import React, { Component } from 'react';

class App extends Component {
	render() {
		return (
			<div className="component-app">
				<h1 className="App-title">React Cards</h1>
			</div>
		);
	}
}

export default App;
