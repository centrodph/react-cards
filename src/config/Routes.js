import React from 'react';
import App from '../pages/App';
import GameConfig from '../pages/GameConfig';

export default [
	{
		path: '/',
		component: App
	},
	{
		path: '/config',
		component: GameConfig
	}
	// {
	// 	path: '/game',
	// 	component: GameWrapper,
	// 	routes: [
	// 		{
	// 			path: '/',
	// 			component: Game
	// 		},
	// 		{
	// 			path: '/result',
	// 			component: GameResult
	// 		}
	// 	]
	// }
];
