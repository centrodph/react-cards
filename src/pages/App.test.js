import React from 'react';
import renderer from 'react-test-renderer';
import { expect as chaiExpect } from 'chai';
import { shallow } from 'enzyme';

import App from './App';
import Routes from '../config/Routes';

describe('Render app', () => {
    // it('check snapshot tree', () => {
    //     const component = renderer.create(<App route={Routes} />);
    //     let tree = component.toJSON();
    //     expect(tree).toMatchSnapshot();
    // });
    it('renders title', () => {
        const wrapper = shallow(<App route={Routes} />);
        chaiExpect(wrapper.find('Header')).to.have.length(1);
    });
    it('check container className component', () => {
        const wrapper = shallow(<App route={Routes} />);
        chaiExpect(wrapper.find('.component-app')).to.have.length(1);
    });
});
