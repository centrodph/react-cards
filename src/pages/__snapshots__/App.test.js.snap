// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render app check snapshot tree 1`] = `
<div
  className="component-app"
>
  <h1
    className="App-title"
  >
    React Cards
  </h1>
</div>
`;
